pkg/sql/copy.go

```diff
@@ -232,6 +232,12 @@ func newCopyMachine(
 
 		c.csvEscape, _ = utf8.DecodeRuneInString(s)
 	}
+	if n.Options.Destination != nil {
+		return nil, pgerror.Newf(
+			pgcode.FeatureNotSupported,
+			"DESTINATION can only be specified when table is external storage table",
```

Comment:
what is "external storage table" / is that clear to the user / should we be attaching an issue to it?

Reply:
Its a fake table name that's indicates a nodelocal or a user file.  I couldn't find any docs on this so I'm not sure how to say it better.  I'm not sure but I think this copy destination form is what `cockroach userfile upload` is implemented with.